### PR TITLE
Refine Learn More sidebars content

### PR DESCRIPTION
**Pull request by 0aveRyan**
These are updates to Learn More sidebars for:
* Welcome
* WP Experience
* Primary Type
* Secondary Type
* Site Features
* What Next

### CODE REVIEW

**Review by officiallygod — APPROVED** [review]

**Review by abhijitb — APPROVED** [review]

File: src/OnboardingSPA/pages/Steps/GetStarted/GetStartedExperience/Sidebar/LearnMore/contents.js [hunk 1]
```diff
@@ -25,7 +21,7 @@ const getContents = ( techSupportLink, fullServiceCreativeTeamLink ) => {
 				{
 					heading: __( 'Why we ask', 'wp-module-onboarding' ),
 					description: __(
-						`We use this to help offer the best WordPress setup, features and suggestions for your site.`,
+						`We want to help everyone get the most out of their WordPress site and this setup, so we use this to help optimize for a great experience and periodically when we offer tips, reminders and recommendations.`,
```
Comment (Yashita101) [inline src/OnboardingSPA/pages/Steps/GetStarted/GetStartedExperience/Sidebar/LearnMore/contents.js:24]: Hey David!
Is the WordPress `site` used intentionally? Or should translations be used?

**Comment by Yashita101** [comment]
**QA** :

https://user-images.githubusercontent.com/53893563/215528593-b3324d05-b416-42f4-961d-0fd1fa862c9c.mp4

File: src/OnboardingSPA/pages/Steps/GetStarted/GetStartedExperience/Sidebar/LearnMore/contents.js (hunk 1 above)
Comment (0aveRyan) [reply src/OnboardingSPA/pages/Steps/GetStarted/GetStartedExperience/Sidebar/LearnMore/contents.js:24]: Yes lets substitute `site` -- good catch!

File: src/OnboardingSPA/pages/Steps/GetStarted/Welcome/Sidebar/LearnMore/contents.js [hunk 2]
```diff
@@ -55,16 +54,17 @@ const getContents = ( brandName, expertsLink, techSupportLink, fullServiceCreati
...
+							`A WordPress %s hosted by %s has tons of unique and proven
+						solutions to help you get farther,
+						faster with your WordPress. We put our expertise, partnerships
+						and solutions to work on your %s.`,
 							'wp-module-onboarding'
 						),
 						translations( 'website' ),
-						brandName
+						brandName,
+						translations('website')
```
Comment (Yashita101) [inline src/OnboardingSPA/pages/Steps/GetStarted/Welcome/Sidebar/LearnMore/contents.js:67]: Hey @0aveRyan , just wanted to confirm the third substitute here - as per the translators mentioned on line 57, it is `site`, but here it is mentioned as `website`. Could you please confirm what is to be used as substitute - `site` or `website`?

Comment (0aveRyan) [reply src/OnboardingSPA/pages/Steps/GetStarted/Welcome/Sidebar/LearnMore/contents.js:67]: @Yashita101 let's go with `site`, thanks!

**Pull request merged**